HL7-DaVinci/test-ehr
# Dmeerx 1142

This PR adds placer identifier for orders when creating the DeviceRequest, ServiceRequest, MedicationRequest and MedicationDispense. 

JIRA ticket: https://jira.mitre.org/browse/DMEERX-1142

To test:
1. create one of the above orders via Postman 
2. verify a placer identifier is added from by checking the response
